apps/vessel-history/features/vessels/activity/vessels-activity.selectors.ts

```diff
@@ -173,6 +206,19 @@ export const selectEventsWithRenderingInfo = createSelector(
             })
             descriptionGeneric = t('event.fishing')
             break
+          case EventTypes.Gap:
+            /*
+            Will split the gaps? 
+            description = event.gap.isEventStart ? t('event.gapStart', 'Gap start in {{regionName}}', {
+              regionName: regionDescription,
+            }) : t('event.gapEnd', 'Gap end in {{regionName}}', {
+              regionName: regionDescription,
+            }) */
```

Comment:
@fertandil87 Could you please remove this commented code block? if we don't need it now it's better to clean this up so the code is more readable and we don't have to think if it is valid, if we have to maintain it, if can or can't be removed...